lib/fuser.h

```diff
@@ -28,22 +29,21 @@ namespace qsim {
 template <typename Gate>
 struct GateFused {
   /**
-   * Kind of the first ("master") gate.
+   * Kind of the first ("parent") gate.
    */
   typename Gate::GateKind kind;
   /**
-   * The time index of the first ("master") gate.
+   * The time index of the first ("parent") gate.
    */
   unsigned time;
-  unsigned num_qubits;
   /**
-   * A list of qubits these gates act upon.
+   * A list of qubits these gates act upon, possibly excluding control qubits.
```

Comment:
Are control qubits only excluded for explicitly-controlled gates (i.e. CX would include its control, but controlled-iSWAP would not)? If so, please note that in the comment here.

Reply:
Done.